object/object.go

```diff
@@ -342,3 +344,39 @@ func (m Macro) Inspect() string {
 	out.WriteString("}")
 	return out.String()
 }
+
+type ExtFunction func(args []Object) Object
+
+type Extension struct {
+	Name     string
+	MinArgs  int
+	MaxArgs  int
+	ArgTypes []Type
+	Callback ExtFunction
+}
+
+func (e *Extension) Usage(out *strings.Builder) {
+	for i := 1; i <= e.MinArgs; i++ {
+		coma := ""
+		if i > 1 {
+			coma = ", "
+		}
+		t := strings.ToLower(e.ArgTypes[i-1].String())
+		out.WriteString(fmt.Sprintf("%s%s%d", coma, t, i))
+	}
+	if e.MaxArgs < 0 {
+		out.WriteString(", ...")
+	} else if e.MaxArgs > e.MinArgs {
+		out.WriteString(fmt.Sprintf(", arg%d...arg%d", e.MinArgs+1, e.MaxArgs))
+	}
```

Comment:
You know I hate else, I cannot help myself reporting them

```suggestion
	if e.MaxArgs < 0 {
		out.WriteString(", ...")
		return
	}
	if e.MaxArgs > e.MinArgs {
		out.WriteString(fmt.Sprintf(", arg%d...arg%d", e.MinArgs+1, e.MaxArgs))
	}
```

Reply:
Funny to find someone who hates `else` even more than me :)
Replacing with
```go
	switch {
	case e.MaxArgs < 0:
		out.WriteString(", ...")
	case e.MaxArgs > e.MinArgs:
		out.WriteString(fmt.Sprintf(", arg%d...arg%d", e.MinArgs+1, e.MaxArgs))
	}
```

Reply:
like it?

Reply:
Yes